Subscriber/ApplePayDirectSubscriber.php

```diff
@@ -14,10 +19,41 @@
 use MollieShopware\Components\Constants\ShopwarePaymentMethod;
 use MollieShopware\Components\Logger;
 use MollieShopware\MollieShopware;
+use Shopware\Components\Model\ModelManager;
 use Shopware\Components\Theme\LessDefinition;
+use Shopware\Models\Customer\Customer;
+use Shopware\Models\Order\Order;
+use Shopware_Components_Config;
+use Shopware_Components_Modules;
 
 class ApplePayDirectSubscriber implements SubscriberInterface
 {
+    const APPLEPAY_NAME = 'mollie_applepay';
```

Comment:
please use constants of class "ShopwarePaymentMethod".
thats already existing

Reply:
Done